src/netbase.cpp

```diff
@@ -749,3 +749,93 @@ void InterruptSocks5(bool interrupt)
 {
     interruptSocks5Recv = interrupt;
 }
+
+bool IsBadPort(uint16_t port)
+{
+    /* Don't forget to update doc/p2p-bad-ports.md if you change this list. */
```

Comment:
Question. Given that this code is nicely commented with ports' corresponding services, and there is a maintenance cost to keeping the lists the same, why does the list need to be duplicated in two places?

Reply:
It is good for the user to be able to see the list of the ports and I think it would be too rough to refer the user to go to look into some .cpp file on line 753.

The other way around would be to generate the .cpp file from the user-facing `p2p-bad-ports.md` but I think that would be an overkill, given that this list is not supposed to be updated often, if ever.

Reply:
Sure, makes sense